src/GameLogic.cpp

```diff
@@ -59,6 +60,20 @@ void GameLogic::pollEvent(sf::RenderWindow *App, sf::Clock gameTime, double targ
 									}
 								}
 								break;
+							case GameState::State::SUCCESS:
+								mousePosition = sf::Mouse::getPosition(*App);
+								if (mousePosition.x >= 0 && mousePosition.x <= 800 && mousePosition.y >= 0 && mousePosition.y <= 600){
+									this -> state.setState(GameState::State::LEVELSELECT);
+								}
+								break;
+							case GameState::State::FAIL:
+								mousePosition = sf::Mouse::getPosition(*App);
+								if (mousePosition.x >= 0 && mousePosition.x <= 800 && mousePosition.y >= 0 && mousePosition.y <= 600){
```

Comment:
remove this for the same reason as above

Reply:
so this case should just be this -> state.setState(GameState::State::LevelSelect) & same as above